### PR TITLE
Support for Multi formatter

### PR DESCRIPTION
**Pull request by gkushang**
#### options.formats
Supports multiple formatter.
Type: `Array`
Available: `['pretty', 'progress', 'summary', 'html']`

e.g. `formats: ['html', 'pretty']`

Note: `html` formatter will provide Json as well as `html` report. Multiple formatter is supported for cucumber v@0.8.0 or higher

### CODE REVIEW

**Comment by gkushang** [comment]
@mavdi Let me know what you think about this? Its keeping backward-compatibility by adding `formats` option, whereas `format` will still work with cucumber v@0.7.0 and lesser.

**Comment by gkushang** [comment]
Hi @mavdi, can you take a look this PR? Thanks!!

File: README.md [hunk 1]
```diff
@@ -108,6 +108,15 @@ Type: `String`
...
 
+#### options.formats
+Supports multiple formatter.
+Type: `Array`
+Available: `['pretty', 'progress', 'summary', 'html']`
+
+e.g. `formats: ['html', 'pretty']`
+
+Note: `html` formatter will provide Json as well as `html` report. Multiple formatter is supported for cucumber v@0.8.0 or higher
+
```
Comment (nikulkarni) [inline README.md]: Should we add a `peerDependency` in package.json to make sure that `cucumber@^0.8.0`? This way we make sure that users do have the correct dependencies prior to using this feature.

**Comment by nikulkarni** [comment]
Otherwise looks good to me :+1:

File: README.md (hunk 1 above)
Comment (gkushang) [inline README.md]: @nikulkarni thanks for reviewing. Multi-formats are not supported for version 0.7.0 and lesser, but still this plugin will still be used. If we add `peerDependency` for cucumber@^0.8.0 then plugin would throw the warning/error. What do you think?

**Pull request merged**